### ISSUE TITLE
Constant Debug Messages when launching Oni2 from CmdLine with Path

### ISSUE THREAD

**Issue by Cy-Tek** [onivim/oni2#1874]
When I launch Oni2 with the command `oni2 --debug -f .`, I get constant messages in the terminal, even when no input is occurring, and the editor itself never fully loads as you can see in the screenshot.

![image](https://user-images.githubusercontent.com/16470547/83782262-91697b80-a65d-11ea-9671-5209d9669f63.png)

Below is some of the output that it keeps repeating:
```
[INFO]     +4ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +5ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +5ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +3ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +1ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +3ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +4ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +2ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +5ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +4ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[INFO]     +0ms Exthost.Transport : Connection closed.
[INFO]     +0ms Oni2.Syntax.Client : Disconnected
[ERROR]    +0ms Oni2.Syntax.Client : Syntax process exited with code: 3221225786 and signal: 0
[INFO]     +0ms Oni2.Store.dispatch : dispatch: (Syntax ServerStopped)
[INFO]     +1ms Oni2.Service_Syntax : Stopping buffer subscription for: 3
[ERROR]    +0ms Oni2.Syntax.Client : Error: Stream.write: broken pipe

[DEBUG]    +0ms Oni2.Store.dispatch : After: (Syntax ServerStopped)
[DEBUG]    +1ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +5ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[DEBUG]    +2ms Oni2.Store.StoreThread : Running effect: Batch:
 explorer.load

[INFO]     +4ms Oni2.Store.dispatch : dispatch: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +1ms Oni2.Store.dispatch : After: (FileExplorer
   (FocusNodeLoaded
      { path = "C:\\Users\\JoshHannaford\\CLionProjects\\meteormd-transpiler";
        displayName = "meteormd-transpiler"; hash = 546755481;
        icon = (Some <opaque>);
        kind = Directory {isOpen = true; children = <opaque>};
        expandedSubtreeSize = 9 }))
[DEBUG]    +0ms Oni2.Store.StoreThread : Effect complete: Batch:
 explorer.load

[INFO]     +0ms Exthost.Transport : Connection closed.
[WARN]     +0ms Exthost.Client : Unhandled message: Exthost_Protocol.Message.Incoming.Disconnected
[INFO]     +0ms Oni2.Extension.ClientStore : Extension host process exited with exit status: 3221225786 and signal: 0
[DEBUG]    +2ms Revery.App : canQuit is true for window 1
[INFO]     +0ms Revery.App : onBeforeQuit
[INFO]     +0ms Oni2.Store.dispatch : dispatch: (Quit true)
[DEBUG]    +0ms Oni2.Store.dispatch : After: (Quit true)
[INFO]     +0ms Revery.App : Quitting
```

**Comment by glennsl** [comment]
I think this is a duplicate of #1799 and fixed in #1808, but the portal builds haven't been working the past few days. It was just fixed though, so tomorrow's build should hopefully see this resolved.

**Comment by bryphe** [comment]
Fixed now with #1808 - please let us know if you're still seeing it, though 👍
